src/player_activity.cpp

```diff
@@ -137,13 +138,13 @@ void player_activity::do_turn( player *p )
             break;
         case ACT_PICKAXE:
             // Based on speed, not time
-            moves_left -= p->moves;
+            moves_left -= (int)(p->moves * season_mult);
```

Comment:
if season_must is not int, just static_cast it, as moves and moves_left are both int.